sdk-blueprints/python/post_process.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+
+
+def process_file(file_path):
+    file_name = os.path.basename(file_path)
+    if not file_name.startswith('test_'):
+        return
+
+    with open(file_path, 'r') as file:
+        orig_content = file.read()
+
+    if "(unittest.TestCase)" not in orig_content or "class Test" not in orig_content:
+        return
+
+    # Replace 'null' with 'None' in the content, as OpenAPI generator uses 'null' instead of 'None' for Python code.
+    content = orig_content.replace('= null', '= None')
```

Comment:
Text literal replacements can have unintended side effects. I assume this is safe?

Reply:
It would be used only for unit tests, which are already broken by the default generation. Even if we have some issues, we should catch them during test execution.